_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,288 @@
+---
+id: b8bb68
+name: Text spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that text spacing set in the `style` attribute is below permissible minimum value or not set to `!important`
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
```

Comment:
I think the idea behind the SC is that enforcing a style that limits the text size and spacing to being smaller than the specified metrics, by using "!important", is bad for accessibility. If the style is already above those metrics it is less impactful to enforce the style. I guess those metrics are a point above which the SC considers content visibility and functionality may be impacted anyway.

Reply:
This bit from the Understanding document might help:

> **Author Responsibility**
>
> This SC does not dictate that authors must set all their content to the specified metrics. Rather, it specifies that an author's content has the ability to be set to those metrics without loss of content or functionality. The author requirement is both to not interfere with a user's ability to override the author settings, and to ensure that content thus modified does not break content in the manners shown in figures 1 through 3 in Effects of Not Allowing for Spacing Override.

This rule checks that the inline styles don't interfere with the user's ability to adjust the text spacing attributes.

Reply:
@kasperisager that's precisely the kind of explanation that can fit into the Background section and help people figure out why this rule is here.
The rule doesn't (directly) checks that "no loss of content occurs when…" but rather that "authors do not block user's style and enforce a bad style" which is related but slightly different.

@EmmaJP 
>  If the style is already above those metrics it is less impactful to enforce the style.

but the rule description (and examples) say that it's OK if the style is **below** the metrics and has `!important` (although the expectations don't).

Reply:
The excerpt above is linked as `Understanding the SC` link in Background section. Please let me know if, and how this can be improved. Suggestions welcome.